test/test_pcaprub_unit.rb

```diff
@@ -164,7 +164,11 @@ def test_filter
 
   def test_lib_version
     v = Pcap.lib_version.split
-    assert_equal "libpcap", v[0]
+    if RUBY_PLATFORM =~ /mingw/
+      assert_equal "WinPcap", v[0]
```

Comment:
This test failed on Windows due to the name difference:

![image](https://user-images.githubusercontent.com/1271782/227414418-944c2a45-bd83-450a-a05e-81b680748f15.png)